plugins/turing/config/config.go

```diff
@@ -31,20 +31,117 @@ type RemoteUI struct {
 }
 
 type ExperimentManagerConfig struct {
-	Enabled        bool           `json:"enabled"`
-	BaseURL        string         `json:"base_url"`      // Base URL for XP experiment REST API
-	HomePageURL    string         `json:"home_page_url"` // Website URL for end-users to manage experiments
-	RemoteUI       RemoteUI       `json:"remote_ui"`
-	RunnerDefaults RunnerDefaults `json:"runner_defaults"`
+	Enabled                      bool                         `json:"enabled"`
+	BaseURL                      string                       `json:"base_url"`      // Base URL for XP experiment REST API
+	HomePageURL                  string                       `json:"home_page_url"` // Website URL for end-users to manage experiments
+	RemoteUI                     RemoteUI                     `json:"remote_ui"`
+	RunnerDefaults               RunnerDefaults               `json:"runner_defaults"`
```

Comment:
Perhaps this change would happen after the plugin's experiment runner has been refactored too. But confirming:
Do we need the `RunnerDefaults` here anymore? Since the endpoint under this struct will no longer be needed with the plugin implementation anymore, shall we remove this struct and simply place the Timeout value under `TreatmentServicePluginConfig `?

Reply:
That's true, I was planning to remove `RunnerDefaults` once I've refactored the plugin's experiment runner, so nope we don't need it anymore once that happens but for now it's still here because the experiment runner's still serving treatments with the endpoint. But yep, once the refactoring has taken place we can probably make the change as you have suggested, 'remove [the] struct and simply place the Timeout value under TreatmentServicePluginConfig'.